### DISCUSSION TITLE
スマートファルコン 完成不了比赛计划

### DISCUSSION THREAD

**Discussion by mtt-konan** [NateScarlet/auto-derby#281] [Q&A] [answered]
它其中一个目标是完成两场pre-op以上比赛1着，正常判断好像不会跑op左右的泥地，但是我用自定义赛程它也不会跑。
不过同是泥地，同样有自由赛程任务的乌拉拉，它就能完成12000人次的目标，不清楚目前能有什么方法可以再提高泥地比赛的评分，从而让它去完成比赛任务

![image](https://user-images.githubusercontent.com/102540842/161382005-6b38ed9a-171b-420c-84f9-4c8e18e27707.png)

**Comment by NateScarlet** [comment, marked as answer]
你试了 example_smart_falcon 插件没

**Reply by mtt-konan** [reply]
没事，我傻了，之前在研究那个插件怎么实现来着
